mycore-base/src/main/java/org/mycore/frontend/servlets/MCRConfigHelperServlet.java

```diff
@@ -0,0 +1,142 @@
+/*
+ *  This file is part of ***  M y C o R e  ***
+ *  See http://www.mycore.de/ for details.
+ *
+ *  MyCoRe is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  MyCoRe is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mycore.frontend.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletOutputStream;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.mycore.common.MCRException;
+import org.mycore.common.config.MCRConfiguration2;
+import org.mycore.common.config.MCRConfigurationException;
+
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Servlet to map the properties defined in the init param "Properties" to a JSON object with the property name as key
+ * and the property value as value. The properties in the init param need to be comma separated and can end with * to
+ * match all properties which start with the string before *.
+ */
+public class MCRConfigHelperServlet extends MCRServlet {
```

Comment:
Extending MCRServlet seems not necessary. No user session, no database transaction. To much overhead. Try extending HttpServlet instead